ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/activityServiceAgent.service.ts

```diff
@@ -126,9 +131,12 @@ export class ActivityServiceAgent extends UserServiceAgent<any> {
             map(httpResponse => httpResponse));
     }
 
-    public deleteActivityInstance(studyGuid: string, activityGuid: string): Observable<any> {
+    public deleteActivityInstance(studyGuid: string, activityGuid: string): Observable<DeleteActivityInstanceResponse | null> {
         const baseUrl = this.getBaseUrl(studyGuid, activityGuid);
-        return this.deleteObservable(baseUrl, null, true);
+
+        return this.deleteObservable(baseUrl, null, true).pipe(
+            map(response => !!response ? response.body as DeleteActivityInstanceResponse : null)
```

Comment:
#minor  map(response => response?.body as DeleteActivityInstanceResponse || null)

Reply:
I just wanted to mirror how `createActivity` method handles this part.